app/models/github_issue.rb

```diff
@@ -19,7 +19,7 @@ def self.fetch(url)
   def self.try_to_get_issue(url)
     client = Octokit::Client.new(access_token: random_token)
     issue = GithubIssue.new(url: url)
-    if !!(url !~ /\/issues\/comments/)
+    if url.match?(/re/)
```

Comment:
Not sure how the regexp turned `/re/`. I would consider adding specs for different cases before modifying this code.
For this pull request, I suggest reverting and adding an exception in the config or for the block of code.

Reply:
Will do. Thanks